src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -225,14 +225,14 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
   });
 
   return (
-    <div className="NavBarWrapper">
+    <div className="NavBarSchedule__wrapper">
       <div className={containerClasses}>
         {/* Disabled as per designs. Up for deletion if confirmied not necessary */}
         {/* {venueId && <ScheduleVenueDescription venueId={venueId} />} */}
 
         <ul className="NavBarSchedule__weekdays">{weekdays}</ul>
         <Toggler
-          containerClassName="NavBarSchedule__bookmarked-toggle"
+          containerClassName="NavBarSchedule__bookmarkedToggle"
```

Comment:
```suggestion
          containerClassName="NavBarSchedule__bookmarked-toggle"
```

Reply:
fixed in 2c16847b0928ba957a737453f7b445db887dda23